src/tools/rustdoc-gui/tester.js

```diff
@@ -70,12 +70,24 @@ function parseOptions(args) {
     return null;
 }
 
+function print_test_successful() {
+    process.stdout.write(".");
+}
+
+function print_test_erroneous() {
+    process.stdout.write("F");
```

Comment:
`console.log` wouldn't do it here?

Reply:
No, because console.log adds a '\n' to each output